unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
+  code
+  name
+  children
+  properties
+}
+
+type DoseStrength {
+  code
+  name
+  children
+  properties
+}
+
+type DoseUnit {
+  code
+  name
+  children
+  properties
+}
+
+type InnerPack {
+  code
+  name
+  children
+  properties
+}
+
+type PackSize {
+  code
+  name
+  children
+  properties
+}
+
+type OuterPack {
```

Comment:
```
export interface IPackOuterEntity extends IEntity {
  code: string;
  name: string;
  type: EEntityType.PackOuter;
  children?: IPackOuterEntityChild[];
  properties?: IPackOuterEntityProperty[];
}
```

Hmm, `PackOuter` or `OuterPack`?

Reply:
I think `OuterPack` sounds better? And also matches the terminology Rowena and Emily are using more closely (which is `Outer Packaging`)

Reply:
Hmm, just thinking for consistency with `PackImmediate` and `PackSize`. `PackOuter` follows the pattern, I guess?

Reply:
Yeah, the consistency is nice. Updated with 5d60a3499 (we currently don't have any codes for this level so just a schema update)